### ISSUE TITLE
It completely breaks Wox

### ISSUE THREAD

**Issue by Francesco270** [inormis/Wox.Plugin.Links#3]
If I install it, it completely breaks Wox, I get no results whatever I write. I had to delete all the plugin files in the Wox folder to make it work again.
I installed it with the command "wpm install Links" and downloading the .wox file from the Wox website. Both ways I still have that bug.

**Comment by inormis** [comment]
Hi @Francesco270 , do you have any logs that could be helpful for investigation?

**Comment by Francesco270** [comment]
> Hi @Francesco270 , do you have any logs that could be helpful for investigation?

Hi, thank you for your reply! 
Sure here it is. I've installed Wox Links at 18:13 then after restart I got no result whatever I 
type.
[2019-12-18.txt](https://github.com/inormis/Wox.Plugin.Links/files/3979659/2019-12-18.txt)

**Comment by inormis** [comment]
Please provide logs from C:\Users\Franc\AppData\Roaming\Wox\Plugins\Wox Links-022618d5-c464-405a-83ae-c83a88452855\logs

**Comment by Francesco270** [comment]
Sorry for providing the wrong file. 
Here is the correct one, it looks like it is almost empty. I've installed it, restarted Wox then it stops working and get no results or suggestions whatever I type (any letter, setting, "link" ecc.).
[Logs20191219.txt](https://github.com/inormis/Wox.Plugin.Links/files/3984276/Logs20191219.txt)

**Comment by djordjedjukic** [comment]
I have the same problem. Log file is the same also.

**Comment by inormis** [comment]
@Francesco270 I apologize for so late reply. Totally forgot about it. Could you tell me if there's a folder C:\Users\Franc\AppData\Roaming\Wox.Plugins.Links\ and if yes, what it has inside.

I wil lcreate shortly new version of Plugin with better logging

**Comment by Francesco270** [comment]
> @Francesco270 I apologize for so late reply. Totally forgot about it. Could you tell me if there's a folder C:\Users\Franc\AppData\Roaming\Wox.Plugins.Links\ and if yes, what it has inside.
> 
> I wil lcreate shortly new version of Plugin with better logging

There is that folder, but it is empty.

If it helps, I've noticed that it doesn't "completely" break Wox, shortcuts of plugins still work, but everything else doesn't.

**Comment by inormis** [comment]
I will create a new version with enhanced logs within 2 days and will come back to you

**Comment by Francesco270** [comment]
> I will create a new version with enhanced logs within 2 days and will come back to you

Hey, I've managed to create a shortcut with the Link plugin, now there are 2 files inside that folder.
Here we go:
[Wox.Plugins.Links.zip](https://github.com/inormis/Wox.Plugin.Links/files/4053381/Wox.Plugins.Links.zip)

And the new log:
[Logs20200113.txt](https://github.com/inormis/Wox.Plugin.Links/files/4053384/Logs20200113.txt)


Also, one last thing, I've tried one of the examples suggested and looks like something is wrong, probably it can't separate correctly path and description
![image](https://user-images.githubusercontent.com/57621306/72254063-4e9dfd00-3603-11ea-8ab6-1f1b95884f6a.png)

**Comment by inormis** [comment]
My bad, I probably haven't updated documentation.
description of the link must be followed after | character. 
`link gh https://github.com | Open GitHub`

**Comment by inormis** [comment]
@Francesco270 any news?

**Comment by Francesco270** [comment]
> @Francesco270 any news?

About the | character? I haven't tried it yet, will do it in about an hour.

**Comment by Francesco270** [comment]
I confirm that with | it works. It is a great plugin, what a pity that it breaks Wox for me.

**Comment by Francesco270** [comment]
Hi, did you manage to make the new version?
Thank you.

**Comment by emirgorodov** [comment]
@inormis actually after installing your plug-in any command with action keyword `*` stop working. All other keywords (`>`, `g`, `wpm`) work just fine. As I can see, the only difference of this plug-in is that it doesn't declare any action keywords.

**Comment by inormis** [comment]
@emirgorodov could you provide list of packages you have installed in WOX?
@Francesco270 I will come back with a new version today

**Comment by inormis** [comment]
@Francesco270 1.2.1 was deployed, please run
`wpm install links`

**Comment by Francesco270** [comment]
> @Francesco270 1.2.1 was deployed, please run
> `wpm install links`

Hi, I've installed it but still it breaks everything except for the shortcuts of plugins.
Here are the file inside the C:\Users\Franc\AppData\Roaming\Wox.Plugins.Links\ folder:
config.json http://bit.ly/2TVHcVO
links.json http://bit.ly/2TYk09B

**Comment by Francesco270** [comment]
If it helps, here is the list of plugins I've installed. It is a huge list, I know..
Bit.ly Shorten URL
Browser Bookmarks
Calculator
Clipboard history
Colors
Control Panel
CurrencyPlus
Dictionary
Direct Translate
Everything
Faver
Folder
Forecast
Giphy
Google Search Plus
Macros
Media
Plugin Indicator
Program
Recall
Reddit Viever
Shell
Simple Web Search
Start Screensaver
System Commands
System Stats
Thesaurus
Timer
Todoist
TodoPlus
ToggleCase
Trabslator
Unit converter
URL
Wheather.Open
Web Searches
WebApp launcher
Wolfram Alpha Query
Wox Plugin Manager
SimpleClock
Volume
Yandex Translate
YouTubeQuery

**Comment by inormis** [comment]
@Francesco270 Please update to 1.2.4

**Comment by Francesco270** [comment]
> @Francesco270 Please update to 1.2.4

Same problem unfortunately.

**Comment by inormis** [comment]
@Francesco270 it was not supposed to fix the issue, but to report about it.
could you tell me what OS do you have?

**Comment by Francesco270** [comment]
> @Francesco270 it was not supposed to fix the issue, but to report about it.
> could you tell me what OS do you have?

So do I have to upload the log?
My OS is Windows 10 home 1903

**Comment by inormis** [comment]
If possible, provide any logs you can find. It looks like it fails before the initialization of the plugin.

**Comment by emirgorodov** [comment]
[2020-03-19.txt](https://github.com/inormis/Wox.Plugin.Links/files/4353381/2020-03-19.txt)

Logs right after start.

**Comment by inormis** [comment]
Links plugin was initialized without any problem
2020-03-19 10:24:04.5378|INFO|PluginManager.InitializePlugins|Total init cost for <Calculator> is <38ms>
2020-03-19 10:24:04.5378|INFO|PluginManager.InitializePlugins|Total init cost for <Colors> is <4ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Everything> is <14ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Folder> is <10ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Plugin Indicator> is <3ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Wox Plugin Management> is <2ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Program> is <21ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <Shell> is <19ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <System Commands> is <2ms>
2020-03-19 10:24:04.5488|INFO|PluginManager.InitializePlugins|Total init cost for <URL> is <4ms>
2020-03-19 10:24:04.5618|INFO|PluginManager.InitializePlugins|Total init cost for <Web Searches> is <60ms>
2020-03-19 10:24:04.7678|INFO|PluginManager.InitializePlugins|Total init cost for <Control Panel> is <232ms>
2020-03-19 10:24:04.9788|INFO|PluginManager.InitializePlugins|Total init cost for <Wox Links> is <426ms>

I see some SSL connection exceptions and smth from Everything plugin.
Could you disable Everything plugin?

**Comment by emirgorodov** [comment]
[2020-03-19.txt](https://github.com/inormis/Wox.Plugin.Links/files/4353604/2020-03-19.txt)

Any changes. 

> Links plugin was initialized without any problem

That's correct, because it's the only one which still works and reacts to any (`*`) keyword.
I still can use `github` or `g` (google) plugins because they have dedicated commands.

**Comment by inormis** [comment]
if you set keyword to l, does it help?

**Comment by emirgorodov** [comment]
![image](https://user-images.githubusercontent.com/8667348/77054677-bb08f400-69d8-11ea-93ff-51c2ce0fec1d.png)
interestingly, I don't have such option for this plugin

**Comment by inormis** [comment]
mhm, ok, I will create a version with a keyword. but IMO for this plugin keyword is redundant.

**Comment by emirgorodov** [comment]
I agree with that, so it should stay configurable. But it may just be the reason why it's broken.

Please, note the difference:
![image](https://user-images.githubusercontent.com/8667348/77308565-d7bc6900-6d03-11ea-8954-2e962be68dc4.png)
![image](https://user-images.githubusercontent.com/8667348/77308310-6ed4f100-6d03-11ea-9d60-bbc0a9fd0754.png)
Both plugins require no keywords to use, but seems to be implemented differently.

**Comment by Francesco270** [comment]
> mhm, ok, I will create a version with a keyword. but IMO for this plugin keyword is redundant.

I think that since there is no keyword to the plugin, everything I type is considered as I'm using the Links plugin. So if I type "setting", for example, it won't display the setting of Wox, but it creates a new Link in the Wox Links shortcut.
